ytmusicapi/parsers/library.py

```diff
@@ -45,6 +45,7 @@ def parse_albums(results):
         data = result[MTRIR]
         album = {}
         album['browseId'] = nav(data, TITLE + NAVIGATION_BROWSE_ID)
+        album['playlistId'] = nav(data, MENU_PLAYLIST_ID)
```

Comment:
`none_if_absent` parameter should be set to true to avoid errors if the menu order is changed or the playlistId is missing for some other reason. This was the case at least once in my tests

Reply:
Good idea! although I would love to get an extract from the json you got in your response in order to try and adapt to this situation.